.github/workflows/server_compatibility.yaml

```diff
@@ -57,6 +57,12 @@ jobs:
           path: hazelcast-enterprise
           ref: ${{ github.event.inputs.branch_name }}
           token: ${{ secrets.GH_PAT }}
+      - name: Checkout to scripts
+        uses: actions/checkout@v2
+      - name: Apply tests-jar diff for 5.2.0
+        if: ${{ github.event.inputs.hz_version == '5.2.0' }}
+        run: git apply $GITHUB_WORKSPACE/tests-jar.diff
+        working-directory: hazelcast-enterprise
```

Comment:
will this work? I mean, the diff file is in the root directory but the working directory is hazelcast-enterprise.

Reply:
Ok sorry it will work I guess

Reply:
we apply `$GITHUB_WORKSPACE/tests-jar.diff` the `$GITHUB_WORKSPACE` contains the proper path to that file